ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -68,6 +68,7 @@ case "${CLUSTER_TYPE}" in
 aws|aws-arm64) export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred;;
 azure4) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json;;
 azurestack) export AZURE_AUTH_LOCATION=${SHARED_DIR}/osServicePrincipal.json;;
+azuremag) export AZURE_AUTH_LOCATION=${SHARED_DIR}/osServicePrincipal.json;;
```

Comment:
After my update for osServicePrincipal.json key in vault, I think we can directly use the file at the default location of $CLUSTER_PROFILE_DIR/osServicePrincipal.json. So it is:
```
azuremag) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json;;
```

Reply:
or just add it after `azure4`
```
azure4|azuremag) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json;;
```